styleguide/webpack.config.js

```diff
@@ -65,6 +66,7 @@ module.exports = {
   },
 
   plugins: [
+    failPlugin,
```

Comment:
Fail plugin doesn't need any extra configuring? Just double checking because all the others do